packages/grafana-data/src/field/fieldOverrides.ts

```diff
@@ -419,10 +422,25 @@ export const getLinksSupplier = (
       },
     };
 
-    info.href = replaceVariables(info.href, variables);
-    info.title = replaceVariables(info.title, variables);
-    info.href = locationUtil.processUrl(info.href);
+    if (link.internal) {
```

Comment:
Main part that makes internal links work in dashboards

Reply:
This part is to be refactored out from here too, right?